pkg/etcdenvvar/etcd_env_test.go

```diff
@@ -0,0 +1,47 @@
+package etcdenvvar
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConvertDBSize(t *testing.T) {
```

Comment:
thanks for adding a unit test here. Can you add another test in TestEnvVarController please?

Reply:
i just checked `TestEnvVarController`, what should i add there ? 

there is no test for the ProfileTuning there 🤔